### ISSUE TITLE
Is this still supported?

### ISSUE THREAD

**Issue by simondsmason** [skarppi/icloud_photo_station#7]
I see that the latest activity on this was Jan 2018.  Is this still supported and working?  Thanks.

**Comment by skarppi** [comment]
The latest release might still work just fine but I don't know for sure. Photo Station part is most probably still good but pyicloud might require upgrading first. Some time ago I tried to merge the latest changes from the upstream repo but it wasn't easy because heavy restructuring of files.

Personally I'm not using Photo Station anymore since it doesn't support HEIF/HEIC files natively and I don't like the idea of converting them back to JPEG. I also found that Photo Station database is not very reliable and sometimes it loses some metadata. However, this might be related to my encrypted disk.

So you might have better luck to use [icloud_photos_downloader](https://github.com/ndbroadbent/icloud_photos_downloader) instead which is more active and can be installed to Synology as a docker image.
